electrumsv/restapi.py

```diff
@@ -0,0 +1,64 @@
+import asyncio
+from typing import Any, Dict, Callable, List
+from aiohttp import web
+import logging
+
+
+class BaseAiohttpServer:
+
+    def __init__(self, host: str = "localhost", port: int = 9999):
+        self.runner = None
+        self.is_alive = False
+        self.app = web.Application()
+        self.app.on_startup.append(self.on_startup)
+        self.app.on_shutdown.append(self.on_shutdown)
+        self.host = host
+        self.port = port
+        self.logger = logging.getLogger("aiohttp REST API")
```

Comment:
Make the log context something lower case and with dashes like "aiohttp-rest-api".